editor-components/modal/notification-modal1/notificationmodal1.tsx

```diff
@@ -0,0 +1,165 @@
+      value: "2 Days remaining",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "description",
+      displayer: "Description",
+      value:
+        "Please pay premium by 05 Dec 2021, otherwise you will be charged 20% amount of your premium.",
+    });
+
+    this.addProp({
+      type: "array",
+      key: "buttonprops",
+      displayer: "Button",
+      additionalParams: {
+        maxElementCount: 2,
+      },
+      value: [
+        {
+          type: "object",
+          key: "item",
+          displayer: "Item",
+          value: [
+            {
+              type: "string",
+              key: "buttonText",
+              displayer: "Button Text",
+              value: "Pay Now",
+            },
+            {
+              type: "page",
+              key: "link",
+              displayer: "Button Link",
+              value: "",
+            },
+          ],
+        },
+        {
+          type: "object",
+          key: "item",
+          displayer: "Item",
+          value: [
+            {
+              type: "string",
+              key: "buttonText",
+              displayer: "Button Text",
+              value: "Visit Page",
+            },
+            {
+              type: "page",
+              key: "link",
+              displayer: "Button Link",
+              value: "",
+            },
+          ],
+        },
+      ],
+    });
+  }
+
+  getName(): string {
+    return "NotificationModal1";
+  }
+
+  render() {
+    const backgroundImage = this.getPropValue("background_image");
+    const overlay = this.getPropValue("overlay");
+
+    return (
+      <div className={this.decorateCSS("container")}>
+        <div className={this.decorateCSS("max-content")}>
+          <div className={this.decorateCSS("content")}>
+            <div className={this.decorateCSS("image-container")}>
+              {backgroundImage && (
+                <>
+                  <img
+                    className={this.decorateCSS("image")}
+                    src={backgroundImage}
+                  />
+                  <h2 className={this.decorateCSS("first-header")}>
+                    {this.getPropValue("firstTitle")}
+                  </h2>
+                </>
+              )}
+              {overlay && (
+                <div className={this.decorateCSS("overlay")}></div>
+              )}
+              {!backgroundImage && overlay && (
+                <div className={this.decorateCSS("overlay2")}>
+                 <h2 className={this.decorateCSS("first-header")}>
+                  {this.getPropValue("firstTitle")}
+                </h2> 
+                </div>
+              )}
+            </div>
+            <div>
+              <h3 className={this.decorateCSS("second-header")}>
+                {this.getPropValue("secondTitle")}
+              </h3>
+              <p className={this.decorateCSS("notice-content")}>
+                {this.getPropValue("description")}
+              </p>
```

Comment:
do not render p if description prop is empty
[check here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)